packages/cli/__tests__/zosjobs/__system__/cancel/cli.zos-jobs.cancel.job.system.test.ts

```diff
@@ -51,6 +55,14 @@ describe("zos-jobs cancel job command", () => {
         it("should surface an error from z/OSMF if the jobid was already canceled", () => {
             runCliScript(__dirname + "/__scripts__/job/submit_job.sh", TEST_ENVIRONMENT, [LOCAL_JCL_FILE]);
             const response = runCliScript(__dirname + "/__scripts__/job/cancel_job_v2_bad.sh", TEST_ENVIRONMENT, [LOCAL_JCL_FILE]);
+            const [, jobname, jobid] = response.stdout.toString().match(jobDataRegex) || [];
+            TEST_ENVIRONMENT.resources.jobData.push({ jobname, jobid });
+
+            // Calculate the previous job ID (one less)
```

Comment:
@t1m0thyj @traeok @awharn @zFernand0 wondering what you guys think about this calculation of a previous jobid. Pretty sure the previous test creates a job that isn't accessible in that test itself.... This previous/"lost" job usually is just one number behind, but I'm sure that's not always the case. Is this okay enough? I'm not sure how to solve the problem of deleting this job otherwise.